test/gtest/conv3d_test.cpp

```diff
@@ -128,19 +128,19 @@ std::vector<TestCase> GetTestCases(const std::string& precision)
     const std::vector<TestCase> test_cases = {
```

Comment:
[<TestCase>](std::vector<**TestCase**>) wont be necessary for this case, you can use std::vector<std::string> instead. Here's an example

test/gtest/conv_hip_igemm_xdlops.cpp